_includes/v23.1/cdc/schema-registry-metric.md

```diff
@@ -0,0 +1 @@
+Use the `changefeed.schema_registry.retry_count` metric to measure the number of request retries performed when sending requests to the schema registry. For more detail on monitoring changefeeds, refer to [Monitor and Debug Changefeeds](monitor-and-debug-changefeeds.html).
```

Comment:
Should this metric also be added to the list of changefeed metrics to track at https://cockroachlabs.com/docs/v23.1/monitor-and-debug-changefeeds.html#recommended-changefeed-metrics-to-track ?

Reply:
Thanks @rmloveland. This is a good question, and one I thought I knew the answer to: that is, we don't have many users of schema registries and so we're not going to dedicate a section to the two metrics we have for it on the monitoring page. Having said that, there's not really a downside to adding a section there, so I'm going to add that. Thanks for prompting me to rethink the approach here!